Userland/SerenityBox/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+serenity_component(
+    SerenityBox
+    REQUIRED
+    TARGETS SerenityBox
+)
+
+set(SOURCES
+    main.cpp
+)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -static")
```

Comment:
I _think_ you can tell CMake that this is a static binary instead of manually passing flags. That might also influence its choice in which libraries it links (assuming that we ever get around to not having `LibCStatic` be a differently named target than `LibC`).